sdks/java/io/solace/src/main/java/org/apache/beam/sdk/io/solace/data/Solace.java

```diff
@@ -248,42 +249,49 @@ public abstract static class Record {
      *
      * @return The attachment data, or an empty ByteString if no attachment is present.
      */
+    @SuppressWarnings("mutable")
     @SchemaFieldNumber("12")
-    public abstract ByteBuffer getAttachmentBytes();
+    public abstract byte[] getAttachmentBytes();
 
-    static Builder builder() {
-      return new AutoValue_Solace_Record.Builder();
+    public static Builder builder() {
+      return new AutoValue_Solace_Record.Builder()
+          .setExpiration(0L)
+          .setPriority(-1)
+          .setRedelivered(false)
+          .setTimeToLive(0)
+          .setAttachmentBytes(ByteString.empty().toByteArray());
```

Comment:
Do we really need to import `org.apache.beam.vendor.grpc.v1p60p1.com.google.protobuf.ByteString` to do this?

Reply:
```suggestion
          .setAttachmentBytes(new byte[0]);
```

Reply:
Fixed in upcoming commit.